include/wlr/render.h

```diff
@@ -32,13 +32,15 @@ struct wlr_texture *wlr_render_texture_create(struct wlr_renderer *r);
  * 	struct wlr_texture *texture;
  * 	float projection[16];
  * 	float matrix[16];
+ * 	float alpha;
  * 	wlr_texture_get_matrix(texture, &matrix, &projection, 123, 321);
- * 	wlr_render_with_matrix(renderer, texture, &matrix);
+ * 	wlr_render_with_matrix(renderer, texture, &matrix, 0.5f);
  *
- * This will render the texture at <123, 321>.
+ * This will render the texture at <123, 321> with an alpha channel of 0.5.
```

Comment:
What's this exactly?
It sounds to me like every alpha value in the texture would be set to 0.5, but judging from changes to the code, I would asume it's actuall half of the original value in the texture.

Reply:
1.0f means texture RGB as is while 0.0 means background shines perfectly through. Try CTRL-shift-a on any window in rootston.

Reply:
Wayland surfaces can be RGBA, not just RGB. Your wording here implies the alpha channel is overwritten when it is in fact a multiplier.